config/.eslintrc.js

```diff
@@ -17,16 +17,6 @@ module.exports = {
       "components": [ "Link" ],
       "specialLink": [ "to" ]
     }],
-    "jsx-a11y/label-has-for": [ 2, {
```

Comment:
This is a deprecated warning.

Reply:
what do you mean? It doesn't apply anymore?

Reply:
To my knowledge it doesn’t get checked anymore and has been replaced by some other rules. At least that’s what I thought I read!